swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=None, item=None):
+        self.inventory = inventory if inventory is not None else []
+        self.item = item
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            for i in range(len(self.inventory)):
+                inventory_set = (self.inventory[i], i)
+                if item in inventory_set:
+                    self.inventory.pop(i)
+                    break
+            return inventory_set[0]
```

Comment:
even though this way works here is a suggestion on how to reduce your lines of code 
```suggestion
            self.inventory.remove(item)
            return item
```